services/121-service/seed-data/program/program-joint-response-ANE.json

```diff
@@ -0,0 +1,380 @@
+{
+  "published": false,
+  "validation": true,
+  "phase": "design",
+  "location": "Ethiopia",
+  "ngo": "ANE",
+  "titlePortal": {
+    "en": "DRA Joint Response 2023 Ethiopia - ANE"
+  },
+  "titlePaApp": {
+    "en": "DRA Joint Response 2023 Ethiopia - ANE"
+  },
+  "description": {
+    "en": ""
+  },
+  "startDate": "2023-08-01T12:00:00Z",
+  "endDate": "2023-12-31T12:00:00Z",
+  "currency": "ETB",
+  "distributionFrequency": "month",
+  "distributionDuration": 3,
+  "fixedTransferValue": 7000,
+  "tryWhatsAppFirst": false,
+  "paymentAmountMultiplierFormula": null,
+  "financialServiceProviders": [
+    {
+      "fsp": "Commercial-bank-ethiopia"
+    }
+  ],
+  "targetNrRegistrations": 250,
+  "meetingDocuments": {
+    "en": ""
+  },
+  "notifications": {
+    "en": {
+      "registered": "This is a message from ANE.\n\nThank you for your registration. We will inform you later if you are included in this project or not."
+    }
+  },
+  "phoneNumberPlaceholder": "+251 000 000 000",
+  "programCustomAttributes": [],
+  "programQuestions": [
+    {
+      "name": "name",
+      "label": {
+        "en": "What is the full name of the head of your household"
+      },
+      "answerType": "text",
+      "questionType": "standard",
+      "options": null,
+      "persistence": true,
+      "export": ["all-people-affected", "included", "selected-for-validation"],
+      "scoring": {},
+      "phases": ["registrationValidation", "inclusion", "payment"],
+      "editableInPortal": true,
+      "shortLabel": {
+        "en": "name"
+      },
+      "duplicateCheck": false,
+      "placeholder": null
+    },
+    {
+      "name": "phone",
+      "label": {
+        "en": "What is your phonenumber?"
+      },
+      "answerType": "numeric",
+      "questionType": "standard",
+      "options": null,
+      "scoring": {},
+      "persistence": true,
+      "pattern": null,
+      "phases": ["registrationValidation", "inclusion", "payment"],
+      "editableInPortal": true,
+      "export": ["all-people-affected", "included", "selected-for-validation"],
```

Comment:
@Vladimir-Petkov In the particular case of phone number, you can leave "phases" array empty, because phoneNumber is already hard-coded in the table, so no need to show it a 2nd time.

Reply:
Make empty phases array for Name and Phone number, because we duplicated them.